ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -52,7 +85,13 @@ public static DriverAction DriveSafelySwitch(LightColor color)
         public static long ForSum(int[] values)
         {
             // TODO: Fix this method so it passes all associated tests. A for loop must be used to calculate the sum.
-            return -1;
+            int summation = 0;
+            for (int i = 0; i < values.Length; i ++)
+            {
+                int x = values[i];
+                summation += x;
```

Comment:
You don't need the x variable here. You can just sum the array value directly:

summation += values[i];